README.md

```diff
@@ -6,3 +6,16 @@ It's built using [Docusaurus](https://docusaurus.io/).
 It also contains a written guide for contributing to Consensys docs, and creating, configuring, and
 testing new doc sites, published at
 [`docs-template.consensys.net`](https://docs-template.consensys.net/).
+
+## Linting
+
+Vale is a work in progress (WIP) for linting: `.github/workflows/lint.yml` uses actions set up on
+Consensys/docs-gha/lint@main.
+
+WIP: That GHA repo should be set up to apply the [Vale linting set](https://github.com/ConsenSys/docs-gha/tree/main/spelling).
+
+### Let's Upset Vale
```

Comment:
Not sure this will upset Vale 
<img width="728" alt="image" src="https://github.com/user-attachments/assets/19460103-f8cc-4dfb-9dc6-2a09664f31eb">
I think it may be constrained to look in the stated folders only and README aint in one!